cicd/gitops/viewing-argo-cd-logs-using-the-kibana-dashboard.adoc

```diff
@@ -0,0 +1,22 @@
+:_content-type: ASSEMBLY
+[id="viewing-argo-cd-logs-using-the-kibana-dashboard"]
+= Viewing Argo CD logs using the Kibana dashboard
+include::_attributes/common-attributes.adoc[]
+:context: viewing-argo-cd-logs-using-the-kibana-dashboard
+
+toc::[]
+
+You can view the Argo CD logs with {logging-title}. The {logging} provides a visualization of logs using Kibana dashboard. To integrate Argo CD with {logging}, the default configuration for the OpenShift Logging Operator enables logging with Argo CD.
+
+[id="prerequisites_viewing-argo-cd-logs-with-the-kibana-dashboard"]
+== Prerequisites
+
+* The {gitops-title} Operator is installed in your cluster.
+* The {logging-title} is installed with default configuration  in your cluster.
```

Comment:
I suggest condering moving these prerequisites section into `modules/gitops-storing-and-retrieving-argo-cd-logs.adoc` as `.Prerequisites` before `.Procedure`.

Reply:
@max-cx is correct. The prereqs (plus the intro paragraph) should be moved to the procdedure module file.